openwisp_controller/config/handlers.py

```
@@ -152,3 +155,30 @@ def organization_disabled_handler(instance, **kwargs):
         # No change in is_active
         return
     tasks.invalidate_controller_views_cache.delay(str(instance.id))
+
+
+def check_ipam_change_handler(sender, instance, **kwargs):
```

Ruff (0.14.14)

[warning] Unused function argument: `kwargs`

(ARG001)